src/docs/product/data-management-settings/dynamic-sampling/example.mdx

```diff
@@ -0,0 +1,110 @@
+writing trace rules that select transactions passing through API 2.0 servers. To select transactions from API-2.0, we
+need to base the rules on the event data.
+
+For targeting specific versions of a software in services that do not initiate a trace, as is the case in our example for
+the API and S1 layers we need to write event based sampling rules. In our case we would write an event rule for
+transactions that select release API 2.0.
+
+**When launching new versions of a system we typically want to temporarily turn up the amount of messages we keep from
+the new system, so we can catch any regressions introduced.**
+
+### Understanding bugs caused by the interaction between multiple systems
+
+After deploying the new version of the mobile app (UI 1.2) and the api (API 2.0) we notice that we are getting errors from
+S1. This is rather unexpected since S1 1.4 has been deployed for a long time and has been operating flawlessly until now.
+
+Looking at errors and transactions in isolation doesn't reveal any pattern.
+To understand what is going on we need to enable sampling by trace information to try to understand exactly in what
+circumstances the error occur. If we sample by trace information, and we have see an error event we know that all the
+transactions from the trace containing the error will also be present.
+
+Here is a sequence of traces that we recorded from which we can get a hint of the problem.
+
+In the table below we Marked with `T` transaction events and with `E` error events.
+In addition to the system and type of message we show information from the trace context (Tr...) and information
+from the (Ev...).
+We have shown a representative trace for all four possible paths a call can go through (T1...T4)
+
+|Nr.|System|Type|Tr. Id|Tr. rel.| Tr. ProjId | Ev Id| Ev rel. |Ev ProjId|
+|: - :|: - :|: - :|: - :|: - :|: - :|: - :|: - :|: - :|
+|1 | UI | T| T1 | UI 1.1 |  p1 | E1 | 1.1 | p1 |
+|2 | API | T | T1 | UI 1.1 | p1 | E2  | API-1.0  | p2  |
+|3 | S1 | T | T1 | UI 1.1 | p1 | E3 | S1-1.4 | p3 |
+|4 | UI | T| T2 | UI 1.1 |  p1 | E4 | 1.1 | p1 |
+|5 | API | T | T2 | UI 1.1 | p1 | E5  | API-2.0  | p2  |
+|6 | S1 | T | T2 | UI 1.1 | p1 | E6 | S1-1.4 | p3 |
+|7 | UI | T| T3 | UI 1.2 |  p1 | E1 | 1.1 | p1 |
+|8 | API | T | T3 | UI 1.2 | p1 | E2  | API-1.0  | p2  |
+|9 | S1 | T | T3 | UI 1.2 | p1 | E3 | S1-1.4 | p3 |
+|10 | S1 | E | T3 | UI 1.2 | p1 | E3 | S1-1.4 | p3 |
+|11 | UI | T| T4 | UI 1.2 |  p1 | E4 | 1.1 | p1 |
+|12 | API | T | T4 | UI 1.2 | p1 | E5  | API-2.0  | p2  |
+|13 | S1 | T | T4 | UI 1.2 | p1 | E6 | S1-1.4 | p3 |
+
+Each trace has 3 transactions, one for each system it passes through.
+
+Looking through a few traces we notice an interesting pattern: only traces that go through UI 1.2, API 1.0 and S1 1.4
+generate errors.
+
+We realise that when we tested the new UI 1.2 we did test the new features with the new API 2.0.
```

Comment:
```suggestion
We realize that when we tested the new UI 1.2 we did test the new features with the new API 2.0.
```

Reply:
both realise and realize are valid spellings. Will leave it as is unless there is a strong preference for one spelling.

Reply:
Go with "realize" - we use US spellings in our docs and that's the more widely used version in the US

Reply:
oh I did not know that..thanks @imatwawana